core/templates/dev/head/css/oppia.css

```diff
@@ -224,15 +197,45 @@ body {
   width: 644px;
 }
 
-@media (max-width: 656px) {
-  .oppia-profile-explorations-container {
-    width: 428px;
+@media (max-width: 800px) {
+  .oppia-profile-container {
+    width: 620px;
+  }
+  .oppia-profile-user-card {
+    width: 180px;
+  }
+  .oppia-profile-content-card {
+    width: 360px;
+  }
+}
+
+@media (max-width: 600px) {
+  .oppia-profile-container {
+    width: 580px;
+  }
+  .oppia-profile-user-card {
+    width: 180px;
+    margin-right: 12px;
+  }
+  .oppia-profile-content-card {
+    width: 360px;
```

Comment:
This plus the user card width cause the right card to fall off the screen at widths between 460 and 600 px.